chart/values.schema.json

```
@@ -1494,5 +1673,5 @@
                 }
             }
         }
-    }
+     }
```

```suggestion
    }
```
@DerekHeldtWerle  Nitpick, don't think this extra space should be here